### PR TITLE
Add documentation and discoverability things

### PR DESCRIPTION
**Pull request by kristleifur**
* add 'repository' and 'readme' fields to pyproject.toml:tool.poetry
* add more info to configuration
* add TODO questions

### CODE REVIEW

**Comment by kristleifur** [comment]
So I put a couple of questions as TODO items in README.md. Maybe you can point me towards the sources of truth on them and I'll complete README.md?

File: README.md [hunk 1]
```diff
@@ -19,15 +19,34 @@ pip install log-enricher
...
+
 Logs will be output in a plain, console-friendly format if `handlers` is 
-`"plain"`, or in a structured JSON format if it is `"structured"`.
\ No newline at end of file
+`"plain"`, or in a structured JSON format if it is `"structured"`.
+
+`initialize_logging()` will talk to and configure the `logging` subsystem. Once that has been called, `logging`
+will use the new configuration.
+
+# TODO: HOW DO ENRICHERS WORK?
```
Comment (arni-inaba) [inline README.md]: Any method returning a dict can be used to enrich log records - https://github.com/arni-inaba/log-enricher/blob/master/log_enricher/enrichers.py#L10-L13

Comment (arni-inaba) [reply README.md]: The key-value pairs in the dict are added as attribute-value pairs to the log record.

Comment (kristleifur) [reply README.md]: Roger! And the methods that those records call, there is no changing log context? - it is just assumed that those methods should work at any time the logging system may log something?

File: README.md [hunk 2]
```diff
@@ -19,15 +19,34 @@ pip install log-enricher
...
+
+# TODO: WHAT OPTIONS ARE ALLOWED IN THE CONFIG DICT?
```
Comment (kristleifur) [inline README.md]: @arni-inaba, so this is a set of things that would be allowed in the config:
```
"handlers": "plain"/"structured",
"log_level": "DEBUG",
"app_version": "xyz",
"release_stage": "staging",
"loggers": ["py", "mylogger"]
```

What values are allowed and what mechanisms do they control?

Comment (kristleifur) [reply README.md]: Ah, it basically prepares things for `logging.dictConfig`, right? https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig

Comment (arni-inaba) [reply README.md]: Précis!
Handlers, loggers and log_level are used that way, app_version and release_stage are used for enrichers.

File: README.md (hunk 1 above)
Comment (arni-inaba) [reply README.md]: Yup! It's the responsibility of the method to return something sensible when called.

**Comment by arni-inaba** [comment]
I would like to merge https://github.com/arni-inaba/log-enricher/pull/7 first and then we can update this one.

**Pull request merged**